nservicebus/upgrades/acs-host-6to7.md

```diff
@@ -8,6 +8,42 @@ related:
 - nservicebus/upgrades/5to6
 ---
 
+## Namespace changes
+
+All types specified in this document are now in the `NServiceBus` namespace.
+
+## IConfigureThisEndpoint changes
+
+`IConfigureThisEndpoint.Customize` is passed an instance of `EndpointConfiguration` instead of `BusConfiguration`.
+
+snippet: AzureServiceBusTransportWithAzureHost
+
+## AsA_Host role changes into IConfigureThisHost
+
+The specifiers `IConfigureThisEndpoint` and `AsA_Host` are now merged into `IConfigureThisHost`. 
```

Comment:
Wait, `IConfigureThisEndpoint` is still there... they are 2 distinct things.
`IConfigureThisEndpoint.Customize(EndpointConfiguration)` 
vs 
`IConfigureThisHost.Configure()` to return host settings